web/src/components/pages/profile/avatar-setup/avatar-setup.tsx

```diff
@@ -96,19 +95,28 @@ interface PropsFromDispatch {
   addNotification: typeof Notifications.actions.addPill;
   refreshUser: typeof User.actions.refresh;
 }
-<MicIcon />;
 
 interface Props extends LocalizationProps, PropsFromState, PropsFromDispatch {}
 
-class AvatarSetup extends React.Component<Props> {
-  state = {
+interface State {
+  isSaving: boolean;
+  isRecording: boolean;
+  isPlaying: boolean;
+  counter: number;
+  clipStatus: 'fetching' | 'notStarted' | 'started' | 'starting' | 'recorded';
+  blobUrl: Blob;
+  avatarClipUrl: string | null;
+}
+
+class AvatarSetup extends React.Component<Props, State> {
+  state: State = {
     isSaving: false,
-    recordingStatus: false,
-    avatarClipPlaying: false,
+    isRecording: false,
+    isPlaying: false,
     counter: 3,
-    clipStatus: 'notStarted',
+    clipStatus: 'fetching',
```

Comment:
By starting in this state we don't have flickering UI caused by a clip being fetched (and the status then changing)